wolfcrypt/src/tfm.c

```diff
@@ -1998,7 +2091,7 @@ static int _fp_exptmod_ct(fp_int * G, fp_int * X, int digits, fp_int * P,
   fp_digit buf, mp;
   int      err, bitcnt, digidx, y;
 
-  /* now setup montgomery  */
+    /* now setup montgomery  */
```

Comment:
does not line up after change in white space

Reply:
good catch. aligned